test/coffee.test.js

```diff
@@ -1,6 +1,6 @@
 // IMPORT MODULES under test here:
 import { renderCoffee } from '../products/product-utils.js';
-import { findByID, calcLineItem, renderLineItem } from '../cart/cart-utils.js';
+import { findByID, calcLineItem, renderLineItem, calcOrderTotal } from '../cart/cart-utils.js';
```

Comment:
great job breaking this up into different branches for different steps, makes it very easy to read and see your process, which is definitely be a good organization skill to have.  Nice job testing